Library/Formula/konoha.rb

```diff
@@ -14,14 +15,13 @@ class Konoha < Formula
   depends_on "json-c"
   depends_on "sqlite"
   depends_on "mecab" if MacOS.version >= :mountain_lion
-  depends_on :python # for python glue code
+  depends_on :python if MacOS.version <= :snow_leopard # for python glue code
 
   def install
-    args = std_cmake_args + [".."]
     mkdir "build" do
-      system "cmake", *args
+      system "cmake", "..", *std_cmake_args
       system "make"
-      system "make", "test" if build.include? "tests"
+      system "make", "test" if build.with?("tests") || build.bottle?
```

Comment:
I find behavior caused by `build.with?("tests") || build.bottle?` can be confusing without reading the code. Why not just enable test by default?